dashboard/test/ui/features/projectsLegacy.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Projects
+
+# TODO - this isnt going to work in other environments (where this channel_id
+#        doesn't exist
+# TODO - IE9 doesnt have push state, and will behave differently
+
+@stop_after_failure
```

Comment:
These tests take an unfortunately long time to run. Each example can wait up to 30 seconds to see the redirect. This becomes particularly painful in cases where something goes wrong. @stop_after_failure makes it so that as soon as we get one failure, we stop running other examples.
Given that + the TODOs, it might make sense to disable this test for now.